feat: Improve status-branch-not-enabled troubleshooting message DOCS-489 (#1554)
codacy/docs: docs/coverage-reporter/index.md

            Coverage was uploaded for a commit that belongs to a branch that isn't analyzed by Codacy.
        </td>
        <td>

>             <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request for the branch so that Codacy starts analyzing the branch automatically (as long as the target branch is enabled on Codacy).</p>

Author: @prcr
I tried to simplify even further. I'm unsure about mentioning that "codacy automatically starts analyzing"--can we reasonably expect that the user takes this outcome for granted?

Let me know if this still conveys the correct and complete meaning.

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request and ensure that the target branch is enabled on Codacy.</p>
```

Reviewer: > [...] can we reasonably expect that the user takes this outcome for granted?

I'm thinking that most users won't immediately make the association between opening a pull request and having the target branch enabled, and Codacy automatically enabling the analysis for the pull request branch. :thinking: 

We could still simplify a bit and avoid the parenthesis, while still including the consequence of Codacy automatically starting to analyze the branch:

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, ensure that the target branch is enabled and open a pull request for Codacy to start analyzing the branch automatically.</p>
```

Author: I like this one!

            Coverage was uploaded for a commit that belongs to a branch that isn't analyzed by Codacy.
        </td>
        <td>
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>.</p>

>             <p>Alternatively, make sure that a pull request for the branch is open and that the target branch is enabled on Codacy.</p>

Author: The PR can also be a draft, but we refer to it as "open".

I guess we can keep the wording; the users can figure this out on their own.

Reviewer: Yes, I agree - we can talk about "opening a pull request" without specifying if it's in the **Draft** or **Open for review** state. :+1:

>             <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>.</p>
>             <p>Alternatively, make sure that a pull request for the branch is open and that the target branch is enabled on Codacy.</p>

Reviewer: A couple of suggestions:

* I propose that we include the two alternative solutions for making sure that Codacy is analyzing the branch in the same paragraph
* The instruction can be more imperative, telling the user to "open a pull request" themselves, instead of making sure that a pull request is open

What do you think?

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request for the branch so that Codacy starts analyzing the branch automatically (as long as the target branch is enabled on Codacy).</p>
```
